### PR TITLE
[MDEP-943] Fix appendOutput option

### PR DESCRIPTION
**Pull request by slawekjaranowski**
https://issues.apache.org/jira/browse/MDEP-943

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/dependency/utils/DependencyUtil.java [hunk 1]
```diff
@@ -226,7 +229,19 @@ public static synchronized void write(String string, File file, boolean append,
             throws IOException {
         Files.createDirectories(file.getParentFile().toPath());
 
-        try (Writer writer = Files.newBufferedWriter(file.toPath(), Charset.forName(encoding))) {
+        OpenOption appendOption;
```
Comment (michael-o) [inline src/main/java/org/apache/maven/plugins/dependency/utils/DependencyUtil.java:232]: Why not make it compact with a ternary operator?

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/dependency/utils/DependencyUtil.java:232]: can be a ternary operator 😄

File: src/test/java/org/apache/maven/plugins/dependency/utils/TestDependencyUtil.java [hunk 2]
```diff
@@ -337,4 +339,28 @@ public void testTokenizer() {
         tokens = DependencyUtil.tokenizer("  ");
         assertEquals(0, tokens.length);
     }
+
+    @Test
+    void outputFileShouldBeOverride() throws IOException {
```
Comment (michael-o) [inline src/test/java/org/apache/maven/plugins/dependency/utils/TestDependencyUtil.java:344]: outputFileShouldBeOverriden

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #1449

**Pull request merged**